flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScan.java

```diff
@@ -88,23 +84,8 @@ public static Object[] parameters() {
     this.fileFormat = FileFormat.valueOf(fileFormat.toUpperCase(Locale.ENGLISH));
   }
 
-  @Before
-  public void before() throws IOException {
```

Comment:
this is refactored to `HadoopCatalogResource` to be reusable (e.g. by the new `TestSqlBase`)